### ISSUE TITLE
"replaceCssVariables" option, is this inverted?

### ISSUE THREAD

**Issue by TomStrepsil** [wuifdesign/postcss-replace-values#3]
Is [this expression](https://github.com/wuifdesign/postcss-replace-values/blob/main/lib/index.ts#L29-L31) perhaps the wrong way round?:
```javascript
if (replaceCssVariables && decl.prop.slice(0, 2) === '--') {
   return;
}
```
It seems like the [default option](https://github.com/wuifdesign/postcss-replace-values/blob/main/lib/index.ts#L15) is to assume `replaceCssVariables === true`, but the [unit test](https://github.com/wuifdesign/postcss-replace-values/blob/main/lib/index.spec.ts#L54-L58) expects no replacement of css variables, despite the implicit option being `true`:
```javascript
  it('should not replace css variable', async () => {
    const source = 'div { --primary: #000; } .btn { color: #000 }';
    const checkValue = await process(source, { values: { '#000': '#a00' } });
    expect(checkValue).toBe('div { --primary: #000; } .btn { color: #a00 }');
  });
```
I would assume this should be:
```javascript
  it('should not replace css variable', async () => {
    const source = 'div { --primary: #000; } .btn { color: #000 }';
    const checkValue = await process(source, { values: { '#000': '#a00' }, replaceCssVariables: false });
    expect(checkValue).toBe('div { --primary: #000; } .btn { color: #a00 }');
  });
```

I have a use case where I would like to replace _within_ a variable declaration, so would like this test:
```javascript
  it('should replace css variable', async () => {
    const source = 'div { --primary: var(--test); }';
    const checkValue = await process(source, { values: { 'var(--test)': '#a00' });
    expect(checkValue).toBe('div { --primary: #a00; }');
  });
```
...to pass.  It does with `replaceCssVariables: false` option - but seems confusing to add the flag this way round.

**Comment by wuifdesign** [comment]
you are right. inverted the check, added your test case and released `v2.0.0`. Should work now (hopefully) ;)

**Comment by TomStrepsil** [comment]
Thanks so much for the swift response!
